include/zephyr/drivers/clock_control/stm32_clock_control.h

```diff
@@ -444,7 +444,7 @@ struct stm32_pclken {
 #define STM32_CLOCK_INFO(clk_index, node_id)				\
 	{								\
 	.enr = DT_CLOCKS_CELL_BY_IDX(node_id, clk_index, bits),		\
-	.bus = DT_CLOCKS_CELL_BY_IDX(node_id, clk_index, bus) & 0xff,	\
+	.bus = DT_CLOCKS_CELL_BY_IDX(node_id, clk_index, bus) & 0x1ff,	\
```

Comment:
Instead of hardcoding the mask, it should be derived from `STM32_CLOCK_DIV_SHIFT` using e.g., `GENMASK` from `zephyr/sys/util.h`:
```suggestion
	.bus = DT_CLOCKS_CELL_BY_IDX(node_id, clk_index, bus) & GENMASK(STM32_CLOCK_DIV_SHIFT - 1, 0)	\
```

Reply:
Indeed, it was even the idea behind introducing `STM32_CLOCK_DIV_SHIFT`, but I somehow missed that place.